src/node_http_parser.cc

```diff
@@ -1037,68 +1037,60 @@ void ConnectionsList::New(const FunctionCallbackInfo<Value>& args) {
 
 void ConnectionsList::All(const FunctionCallbackInfo<Value>& args) {
   Isolate* isolate = args.GetIsolate();
-  Local<Context> context = isolate->GetCurrentContext();
 
-  Local<Array> all = Array::New(isolate);
   ConnectionsList* list;
 
   ASSIGN_OR_RETURN_UNWRAP(&list, args.Holder());
 
-  uint32_t i = 0;
+  std::vector<Local<Value>> result;
```

Comment:
What about using `MaybeStackBuffer` to avoid allocations if number of elements is not too high?

Reply:
I think `v8::Array` -> `std::vector` would make sense (to avoid C++ -> JS invocation costs) but `std::vector` -> `MaybeStackBuffer` probably would not make much difference...